tests/cppunit/status_test.cc

```diff
@@ -26,7 +26,7 @@
 TEST(StatusOr, Scalar) {
   auto f = [](int x) -> StatusOr<int> {
     if (x > 10) {
-      return {Status::NotOK, "x large than 10"};
+      return Status::NotOK("x large than 10");
```

Comment:
Actually the original code `{Status::NotOK, "x large than 10"}` here is typed `StatusOr<int>`, i.e. it equals `StatusOr<int>{Status::NotOK, "x large than 10"}`, and the new code `Status::NotOK("x large than 10")` is typed `Status`. Although since we have a constructor `StatusOr<T>(Status)` so it will go well.